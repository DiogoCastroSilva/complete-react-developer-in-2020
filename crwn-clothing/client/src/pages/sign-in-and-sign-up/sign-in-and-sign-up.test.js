import React from 'react';
import { shallow } from 'enzyme';

import SignInAndSignUp from './sign-in-and-sign-up.component';


describe('testing SignInAndSignUp component', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<SignInAndSignUp />);
    });

    it('should render a SignInAndSignUp component', () => {
        expect(wrapper).toMatchSnapshot();
    });
});