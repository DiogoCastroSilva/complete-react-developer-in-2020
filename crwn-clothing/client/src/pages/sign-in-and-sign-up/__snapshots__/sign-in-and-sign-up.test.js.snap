// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing SignInAndSignUp component should render a SignInAndSignUp component 1`] = `
<styled.div>
  <Connect(SignInContainer) />
  <Connect(SignUpContainer) />
</styled.div>
`;
